Libs/MRML/Widgets/qMRMLTransformSliders.cxx

```diff
@@ -458,19 +458,19 @@ void qMRMLTransformSliders::resetUnactiveSliders()
 {
   Q_D(qMRMLTransformSliders);
 
-  if (!d->ActiveSliders.contains(d->LRSlider))
+  if (!d->ActiveSliders.contains(d->LRSlider) && !d->LRSlider->isSettingValueFromSpinBox())
```

Comment:
isn't `QObject::sender()` sufficient ? (I did not think of it carefully)

Reply:
The direct sender is not that we need, because the series of events looks like this:
- user types one more digit into spinbox A
- number of digits is increased, so spinbox A emits a signal
- spinbox B (sibling widget of A) receives the signal that number of digits are modified and updates its value (the value is determined from slider B, rounded to the new number of digits)
- spinbox B value change triggers reset of all sliders/spinboxes to 0
- we should prevent reset of spinbox A; but the direct sender of the event is spinbox B